02_configure_host.sh

```diff
@@ -243,3 +243,13 @@ if [ "$DISABLE_MULTICAST" == "true" ]; then
         sudo ebtables -A OUTPUT --pkttype-type multicast -p ip6 --ip6-dst ${dst} -j DROP
     done
 fi
+
+# Hack to force use of infinite leases in versions of libvirt that don't
+# support setting the lease time in the network config.
+# This should be removed when we have a new enough version of libvirt to
+# do it properly.
+if [ ${INFINITE_LEASES:-0} -eq 1 ]
+then
+    ./infinite-leases.sh &
+    echo "$!" > infinite-lease-pid
```

Comment:
It'd probably be better to put the pid file into `$WORKING_DIR` I think?

https://github.com/openshift-metal3/dev-scripts/blob/master/common.sh#L72

Reply:
Yep, good point.